2021-11-18 23:18:06,177 INFO    MainThread:12188 [wandb_setup.py:_flush():71] setting env: {}
2021-11-18 23:18:06,177 INFO    MainThread:12188 [wandb_setup.py:_flush():71] setting login settings: {'anonymous': 'must'}
2021-11-18 23:18:06,178 INFO    MainThread:12188 [wandb_init.py:_log_setup():357] Logging user logs to /home/nicolas/workspace_josue/Pytorch-UNet/wandb/run-20211118_231806-33zl8o7q/logs/debug.log
2021-11-18 23:18:06,178 INFO    MainThread:12188 [wandb_init.py:_log_setup():358] Logging internal logs to /home/nicolas/workspace_josue/Pytorch-UNet/wandb/run-20211118_231806-33zl8o7q/logs/debug-internal.log
2021-11-18 23:18:06,179 INFO    MainThread:12188 [wandb_init.py:init():390] calling init triggers
2021-11-18 23:18:06,179 INFO    MainThread:12188 [wandb_init.py:init():397] wandb.init called with sweep_config: {}
config: {}
2021-11-18 23:18:06,179 INFO    MainThread:12188 [wandb_init.py:init():435] starting backend
2021-11-18 23:18:06,179 INFO    MainThread:12188 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-11-18 23:18:06,223 INFO    MainThread:12188 [backend.py:ensure_launched():199] starting backend process...
2021-11-18 23:18:06,243 INFO    MainThread:12188 [backend.py:ensure_launched():205] started backend process with pid: 12208
2021-11-18 23:18:06,244 INFO    MainThread:12188 [wandb_init.py:init():444] backend started and connected
2021-11-18 23:18:06,246 INFO    MainThread:12188 [wandb_init.py:init():503] updated telemetry
2021-11-18 23:18:06,295 INFO    MainThread:12188 [wandb_init.py:init():533] communicating current version
2021-11-18 23:18:06,611 INFO    MainThread:12188 [wandb_init.py:init():538] got version response 
2021-11-18 23:18:06,611 INFO    MainThread:12188 [wandb_init.py:init():548] communicating run to backend with 30 second timeout
2021-11-18 23:18:06,945 INFO    MainThread:12188 [wandb_init.py:init():576] starting run threads in backend
2021-11-18 23:18:07,884 INFO    MainThread:12188 [wandb_run.py:_console_start():1693] atexit reg
2021-11-18 23:18:07,884 INFO    MainThread:12188 [wandb_run.py:_redirect():1567] redirect: SettingsConsole.REDIRECT
2021-11-18 23:18:07,885 INFO    MainThread:12188 [wandb_run.py:_redirect():1572] Redirecting console.
2021-11-18 23:18:07,886 INFO    MainThread:12188 [wandb_run.py:_redirect():1628] Redirects installed.
2021-11-18 23:18:07,886 INFO    MainThread:12188 [wandb_init.py:init():603] run started, returning control to user process
2021-11-18 23:18:07,887 INFO    MainThread:12188 [wandb_run.py:_config_callback():911] config_cb None None {'epochs': 5, 'batch_size': 5, 'learning_rate': 1e-05, 'val_percent': 0.1, 'save_checkpoint': True, 'img_size': [250.0, 250.0], 'amp': True}
2021-11-18 23:18:08,125 INFO    MainThread:12188 [wandb_run.py:_atexit_cleanup():1663] got exitcode: 1
2021-11-18 23:18:08,125 INFO    MainThread:12188 [wandb_run.py:_restore():1635] restore
2021-11-18 23:18:10,956 INFO    MainThread:12188 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 862
  total_bytes: 862
}

2021-11-18 23:18:11,156 INFO    MainThread:12188 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 862
  total_bytes: 862
}

2021-11-18 23:18:11,964 INFO    MainThread:12188 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 862
  total_bytes: 15604
}

2021-11-18 23:18:12,067 INFO    MainThread:12188 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 862
  total_bytes: 16331
}

2021-11-18 23:18:12,169 INFO    MainThread:12188 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16331
  total_bytes: 16331
}

2021-11-18 23:18:12,272 INFO    MainThread:12188 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16331
  total_bytes: 16331
}

2021-11-18 23:18:12,375 INFO    MainThread:12188 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16331
  total_bytes: 16331
}

2021-11-18 23:18:12,477 INFO    MainThread:12188 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16331
  total_bytes: 16331
}

2021-11-18 23:18:12,581 INFO    MainThread:12188 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16331
  total_bytes: 16331
}

2021-11-18 23:18:12,803 INFO    MainThread:12188 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16331
  total_bytes: 16331
}

2021-11-18 23:18:13,195 INFO    MainThread:12188 [wandb_run.py:_wait_for_finish():1793] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16331
  total_bytes: 16331
}
local_info {
}

2021-11-18 23:18:14,253 INFO    MainThread:12188 [wandb_run.py:_append_files():2061] logging synced files
